app/ui/Home.py

```diff
@@ -1,46 +1,206 @@
+        lambda x: datetime.datetime.strptime(x, "%Y-%m-%d %H:%M:%S.%f %z")
+        if x is not None
+        else None,
+    )
+    qst = get(
+        queryh_start_times,
+        lambda x: x.replace(tzinfo=qet.tzinfo)
+        if x is not None and qet is not None
+        else None,
+    )
+    wst = get(
+        warehouse_start_times,
+        lambda x: x.replace(tzinfo=wet.tzinfo)
+        if x is not None and wet is not None
+        else None,
+    )
+    q1, q2 = to_str(qet, qst)
+    w1, w2 = to_str(wet, wst)
+    rmin = get(
+        range_min, lambda x: x.strftime("%Y-%m-%d %H:%M") if x is not None else None
+    )
+    rmax = get(
+        range_max, lambda x: x.strftime("%Y-%m-%d %H:%M") if x is not None else None
     )
-else:
+    return q1, q2, w1, w2, rmin, rmax
+
+
+def get_wh_utilization():
+    query = """select iff(sum(loaded_cc) = 0,null, sum(unloaded_cc)/sum(loaded_cc)) as utilization
+        from reporting.warehouse_daily_utilization
+        where period::DATE between current_timestamp - interval '30 days' and current_timestamp """
+    return get(query, lambda x: f"{x*100:.2f} %" if x is not None else "Calculating")
+
+
+cols = st.columns([30, 20])
+with cols[0]:
+    sthelp.image_svg("opscenter_logo.svg")
     st.markdown(
         """
-            Run the following command in snowsight to set up your Sundeck account and get started:
-            ```
-            begin
-                var setup_script varchar;
-                call sundeck_opscenter.admin.register() into :setup_script;
-                execute immediate :setup_script;
-            end;
-            ```
+    The Sundeck native app is part of the Sundeck query engineering platform.
+    The native app is designed to give you better insight into your Snowflake
+    consumption and provide a useful set of tools to improve behavior.
+    """
+    )
+
+    wh = get_wh_utilization()
+    st.metric("Warehouse Utilization", wh)
+    if wh == "Calculating":
+        st.markdown(
             """
+Note: Upon installation, the Sundeck native app analyzes your Snowflake usage
+history. Depending on the size of this data, this may takes minutes to hours.
+Once complete, utilization will reported above. (this is only shown until
+first materialization
+        """
+        )
+    st.markdown(
+        """
+
+## Enhanced Snowflake Consumption Tables
+The Sundeck native app automatically generates a fully materialized and incrementally
+updated set of warehouse and query activity data tables. This allows you to analyze
+key `SNOWFLAKE.ACCOUNT_USAGE` tables efficiently. These tables also are enriched to
+enhance analysis opportunities. Key tables include:
+
+#### REPORTING.ENRICHED_QUERY_HISTORY
+An enriched variation of `SNOWFLAKE.ACCOUNT_USAGE.QUERY_HISTORY`. Includes all
+columns from that tables as well as per query estimated costs using Sundeck's cost
+attribution model and automatically extracted QTag values. Additional details can be
+found here.
```

Comment:
> can be found here

Should the "here" words in this paragraph and below be hyperlinks to Sundeck docs?

Reply:
added a few, waiting on feedback from J for the rest